pkg/cmd/target.go

```diff
@@ -543,6 +507,43 @@ func targetShoot(targetWriter TargetWriter, shoot v1beta1.Shoot) {
 
 	KUBECONFIG = shootKubeconfigPath
 	fmt.Println("KUBECONFIG=" + KUBECONFIG)
+
```

Comment:
Moving this code to the end of the function causes the target cmd to fail, please undo this change.